cob_cartesian_controller/scripts/lwa4d/lwa4d_quad_angular.py

```diff
@@ -0,0 +1,121 @@
+#! /usr/bin/env python
```

Comment:
in the end, I think we should remove the lwa4d_* scripts from cob_cartesian_controller and put them into `schunk_lwa4d` directly!